contracts/payloads/IGP100/PayloadIGP100.sol

```diff
@@ -314,6 +317,174 @@ contract PayloadIGP100 is PayloadIGPMain {
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+
+            {
+                address wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT = getVaultAddress(
+                        138
+                    );
+                address USDC_USDT_CONCENTRATED_DEX = getDexAddress(34);
+                address wstUSR_USDC_DEX = getDexAddress(27);
+
+                {
+                    // Update wstUSR-USDC<>USDC-USDT concentrated vault supply shares limit
+                    IFluidAdminDex.UserSupplyConfig[]
+                        memory config_ = new IFluidAdminDex.UserSupplyConfig[](
+                            1
+                        );
+                    config_[0] = IFluidAdminDex.UserSupplyConfig({
+                        user: wstUSR_USDC__USDC_USDT_VAULT,
+                        expandPercent: 35 * 1e2, // 25%
+                        expandDuration: 6 hours, // 12 hours
+                        baseWithdrawalLimit: 5_000 * 1e18 // 5k shares (10k)
+                    });
+
+                    IFluidDex(wstUSR_USDC_DEX).updateUserSupplyConfigs(config_);
+                }
+
+                {
+                    // Update wstUSR-USDC<>USDC-USDT concentrated vault borrow shares limit
+                    IFluidAdminDex.UserBorrowConfig[]
+                        memory config_ = new IFluidAdminDex.UserBorrowConfig[](
+                            1
+                        );
+                    config_[0] = IFluidAdminDex.UserBorrowConfig({
+                        user: wstUSR_USDC__USDC_USDT_VAULT,
+                        expandPercent: 30 * 1e2, // 20%
+                        expandDuration: 6 hours, // 12 hours
+                        baseDebtCeiling: 4_000 * 1e18, // 4k shares ($8k)
+                        maxDebtCeiling: 5_000 * 1e18 // 5k shares ($10k)
+                    });
+
+                    IFluidDex(USDC_USDT_CONCENTRATED_DEX)
+                        .updateUserBorrowConfigs(config_);
+                }
+
+                VAULT_FACTORY.setVaultAuth(
+                    wstUSR_USDC__USDC_USDT_CONCENTRATED_VAULT,
+                    TEAM_MULTISIG,
+                    true
+                );
+            }
+        }
+    }
+
+    // @notice Action 8: Set dust limits for GHO-USDe DEX
+    function action8() internal isActionSkippable(8) {
+        {
+            // GHO-USDe DEX
+            DexConfig memory DEX_GHO_USDe = DexConfig({
+                dex: GHO_USDe_DEX,
+                tokenA: GHO_ADDRESS,
+                tokenB: USDe_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_GHO_USDe); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(GHO_USDe_DEX, TEAM_MULTISIG, true);
+        }
```

Comment:
+ borrow dust limit for T4 vault on GHO USDC + make team ms an auth on the new vault